app/assets/javascripts/angular-libs/angular.of.directives.js

```diff
@@ -16,6 +16,16 @@ openFarmModule.directive('markdown', ['$sanitize',
     };
 }]);
 
+openFarmModule.directive('loader', ['$rootScope',
+  function loader($rootScope) {
+    return {
+      restrict: 'A',
+      link: function (scope, element, attrs) {
```

Comment:
'attrs' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.